.editorconfig

```diff
@@ -83,6 +83,7 @@ dotnet_style_readonly_field = true:suggestion
 
 # Parameter preferences
 dotnet_code_quality_unused_parameters = all:warning
+dotnet_code_quality.ca2208.api_surface = public
```

Comment:
Removes suggestion of using `paramName` on non-public methods.

Rationale: validation is `internal`, therefore sometimes it's more helpful to know the type name compared to the locally-named variable.

```csharp
internal static class Ensure
{
    public static void Valid(AvailableRoomTypesRequest message) // "message" is an internal variable and Type
    {
        if (message is null)
            throw new ArgumentNullException(nameof(AvailableRoomTypesRequest));

            // Without CA2208 override, the IDE would suggest the following:
            // throw new ArgumentNullException(nameof(message));
            // This is less helpful.
    }
}
```